### PR TITLE
[zh] translate update-with-ingress-nginx page

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/36344

https://kubernetes.io/blog/2021/07/26/update-with-ingress-nginx/ does not have chinese version

so need to translate it into Chinese

### CODE REVIEW

**Comment by twilight0620** [comment]
I have modified them. Thanks for your review.

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+<!--
+During community meetings, the networking Special Interest Group has decided to continue 
+supporting Kubernetes versions older than 1.22 with Ingress-NGINX version 0.47.0. 
+Support for Ingress-NGINX will continue for six months after Kubernetes 1.22 
+is released. Any additional bug fixes and CVEs for Ingress-NGINX will be 
+addressed on a need-by-need basis.
+-->
+在社区会议中，网络特别兴趣小组决定继续支持带有 0.47.0 版本 Ingress-NGINX 的早于 1.22 版本的 Kubernetes。
+在 Kubernetes 1.22 发布后，对 Ingress-NGINX 的支持将持续六个月。
+根据需要解决 Ingress-NGINX 的错误修复和 CVE。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:65]:
```suggestion
团队会根据需要解决 Ingress-NGINX 的额外错误修复和 CVE 问题。
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+<!--
+Ingress-NGINX will have separate branches and releases of Ingress-NGINX to 
+support this model, mirroring the Kubernetes project process. Future 
+releases of the Ingress-NGINX project will track and support the latest 
+versions of Kubernetes.
+-->
+Ingress-NGINX 将拥有独立的分支和版本来支持该模型，与 Kubernetes 项目流程相一致。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:73]:
```suggestion
Ingress-NGINX 将拥有独立的分支和发布版本来支持这个模型，与 Kubernetes 项目流程相一致。
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+# 你需要做什么
+
+<!--
+The team is currently in the process of upgrading ingress-nginx to support 
+the v1 migration, you can track the progress 
+[here](https://github.com/kubernetes/ingress-nginx/pull/7156).  
+We're not making feature improvements to `ingress-nginx` until after the support for
+Ingress v1 is complete.
+-->
+团队目前正在升级 Ingress-NGINX 以支持 v1 迁移，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:113]:
```suggestion
团队目前正在升级 Ingress-NGINX 以支持向 v1 的迁移，
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+你可以在[此处](https://github.com/kubernetes/ingress-nginx/pull/7156)跟踪进度。
+在对 Ingress v1 的支持完成之前，
+我们不会对功能进行改进。
+
+<!--
+In the meantime to ensure no compatibility issues: 
+-->
+同时确保没有兼容性问题：
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:121]:
```suggestion
同时，团队会确保没有兼容性问题：
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+<!--
+The community’s feedback and support in this effort is welcome. The
+Ingress-NGINX Sub-project regularly holds community meetings where we discuss
+this and other issues facing the project. For more information on the sub-project, 
+please see [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+-->
+欢迎社区对此工作的反馈和支持。
+Ingress-NGINX 子项目定期举行社区会议，
+我们讨论该问题以及项目面临的其他问题。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:148]:
```suggestion
我们会讨论这个问题以及项目面临的其他问题。
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+-->
+**作者：** James Strong, Ricardo Katz
+
+<!--
+With all Kubernetes APIs, there is a process to creating, maintaining, and
+ultimately deprecating them once they become GA. The networking.k8s.io API group is no
+different. The upcoming Kubernetes 1.22 release will remove several deprecated APIs
+that are relevant to networking:
+-->
+对于所有 Kubernetes API，一旦它们成为 GA，就有一个创建、维护和最终弃用它们的过程。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:25]:
```suggestion
对于所有 Kubernetes API，一旦它们被正式发布（GA），就有一个创建、维护和最终弃用它们的过程。
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+-->
+- [IngressClass](/zh-cn/docs/concepts/services-networking/ingress/#ingress-class) 的 `networking.k8s.io/v1beta1` API 版本
+- [Ingress](/zh-cn/docs/concepts/services-networking/ingress/) 的所有 beta 版本: `extensions/v1beta1` 和 `networking.k8s.io/v1beta1`
+
+<!--
+On a v1.22 Kubernetes cluster, you'll be able to access Ingress and IngressClass
+objects through the stable (v1) APIs, but access via their beta APIs won't be possible.
+-->
+在 v1.22 Kubernetes 集群上，你能够通过稳定（v1）API 访问 Ingress 和 IngressClass 对象，
+但无法通过它们的 Beta API 访问。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:41]:
```suggestion
但无法通过其 Beta API 访问。
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+- [IngressClass](/zh-cn/docs/concepts/services-networking/ingress/#ingress-class) 的 `networking.k8s.io/v1beta1` API 版本
+- [Ingress](/zh-cn/docs/concepts/services-networking/ingress/) 的所有 beta 版本: `extensions/v1beta1` 和 `networking.k8s.io/v1beta1`
+
+<!--
+On a v1.22 Kubernetes cluster, you'll be able to access Ingress and IngressClass
+objects through the stable (v1) APIs, but access via their beta APIs won't be possible.
+-->
+在 v1.22 Kubernetes 集群上，你能够通过稳定（v1）API 访问 Ingress 和 IngressClass 对象，
+但无法通过其 Beta API 访问。
+<!--
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:42]: Add a newline between paragraphs.

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+This change has been in
+in discussion since
+[2017](https://github.com/kubernetes/kubernetes/issues/43214),
+[2019](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) with 
+1.16 Kubernetes API deprecations, and most recently in
+KEP-1453: 
+[Graduate Ingress API to GA](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1453-ingress-api#122).
+-->
+自 [2017](https://github.com/kubernetes/kubernetes/issues/43214),
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:51]:
```suggestion
自 [2017](https://github.com/kubernetes/kubernetes/issues/43214)、
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+<!--
+In the meantime to ensure no compatibility issues: 
+-->
+同时，团队会确保没有兼容性问题：
+
+<!--
+* Update to the latest version of Ingress-NGINX; currently
+  [v0.47.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v0.47.0) 
+-->
+* 更新到最新的 Ingress-NGINX 版本;
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:127]: We can use `；` or `，` for English `; `.
```suggestion
* 更新到最新的 Ingress-NGINX 版本，
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+<!--
+* After Kubernetes 1.22 is released, ensure you are using the latest version of 
+  Ingress-NGINX that supports the stable APIs for Ingress and IngressClass.
+-->  
+* Kubernetes 1.22 发布后，请确保使用的是支持 Ingress 和 IngressClass 稳定 API 的最新版本的 Ingress-NGINX。
+<!--
+* Test Ingress-NGINX version v1.0.0-alpha.2 with Cluster versions >= 1.19 
+  and report any issues to the projects Github page. 
+-->
+* 使用集群版本 >= 1.19 测试 Ingress-NGINX 版本 v1.0.0-alpha.2，并将任何问题报告给项目 Github 页面。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:138]:
```suggestion
* 使用集群版本 >= 1.19 测试 Ingress-NGINX 版本 v1.0.0-alpha.2，并将任何问题报告给项目 GitHub 页面。
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md (hunk 8 above)
Comment (twilight0620) [reply content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:42]: They are in one paragraph, it is the same pattern with the en docs

**Comment by twilight0620** [comment]
does anybody handle this pr？

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+**作者：** James Strong, Ricardo Katz
+
+<!--
+With all Kubernetes APIs, there is a process to creating, maintaining, and
+ultimately deprecating them once they become GA. The networking.k8s.io API group is no
+different. The upcoming Kubernetes 1.22 release will remove several deprecated APIs
+that are relevant to networking:
+-->
+对于所有 Kubernetes API，一旦它们被正式发布（GA），就有一个创建、维护和最终弃用它们的过程。
+network.k8s.io API 组也不例外。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:26]:
```suggestion
networking.k8s.io API 组也不例外。
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 13]
```diff
@@ -0,0 +1,149 @@
...
+即将发布的 Kubernetes 1.22 版本将删除几个与网络相关的已弃用 API：
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:27]:
```suggestion
即将发布的 Kubernetes 1.22 版本将移除几个与网络相关的已弃用 API：
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+
+<!--
+- the `networking.k8s.io/v1beta1` API version of [IngressClass](/docs/concepts/services-networking/ingress/#ingress-class)
+- all beta versions of [Ingress](/docs/concepts/services-networking/ingress/): `extensions/v1beta1` and `networking.k8s.io/v1beta1`
+-->
+- [IngressClass](/zh-cn/docs/concepts/services-networking/ingress/#ingress-class) 的 `networking.k8s.io/v1beta1` API 版本
+- [Ingress](/zh-cn/docs/concepts/services-networking/ingress/) 的所有 beta 版本: `extensions/v1beta1` 和 `networking.k8s.io/v1beta1`
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:34]:
```suggestion
- [Ingress](/zh-cn/docs/concepts/services-networking/ingress/) 的所有 Beta 版本: `extensions/v1beta1` 和 `networking.k8s.io/v1beta1`
```

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 15]
```diff
@@ -0,0 +1,149 @@
...
+
+<!--
+On a v1.22 Kubernetes cluster, you'll be able to access Ingress and IngressClass
+objects through the stable (v1) APIs, but access via their beta APIs won't be possible.
+-->
+在 v1.22 Kubernetes 集群上，你能够通过稳定（v1）API 访问 Ingress 和 IngressClass 对象，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:40]:
```suggestion
在 v1.22 Kubernetes 集群上，你能够通过稳定版本（v1）的 API 访问 Ingress 和 IngressClass 对象，
```

**Comment by windsonsea** [comment]
> I have modified them. Thanks for your review.

We hope you could read and check carefully after 3 rounds review.

File: content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+<!--
+Because of the updates in Kubernetes 1.22, **v0.47.0** will not work with 
+Kubernetes 1.22. 
+-->
+由于 Kubernetes 1.22 中的更新，**v0.47.0** 将无法与 Kubernetes 1.22 一起使用。 
+
+<!--
+# What you need to do
+-->
+# 你需要做什么
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md:104]:
```suggestion
## 你需要做什么
```
The en upstream is not correct.

**Pull request merged**